source/administration/config-settings.rst

```diff
@@ -3732,6 +3740,19 @@ Changes made when hardened mode is enabled:
 | This feature’s ``config.json`` setting is ``"ExperimentalEnableHardenedMode": false`` with options ``true`` and ``false`` for above settings respectively.          |
 +---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Limit Access to Client Config Settings Prior to Login
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enable this setting to limit the number of client config.json settings prior to login.
```

Comment:
Maybe `Enable this setting to limit the number of config settings sent to users prior to login.`? That might